GCMessengerSDKSample/app/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
             oktaProperties.load(oktaPropertiesFile.newReader())
             println("okta.properties loaded.")
         } else {
-            println("okta.properties does not exist in the path and will be ignored.")
+            throw new GradleException("okta.properties does not exist in the path (${oktaPropertiesFile.absolutePath}) and will be ignored.")
```

Comment:
"and will be ignored" - this part of the message doesn’t make much sense now.

Do we have some default okta.properties which allows us to build app after git repo checkout?

If not it means we need to provide instructions how to get okta.properties in order to build application or bypass this exception if we want to build test app without authentication enabled.